build/assets/js/1f2dc99a.676c8cca.js

```
@@ -0,0 +1 @@
+"use strict";(self.webpackChunkmorph_doc=self.webpackChunkmorph_doc||[]).push([[8170],{3306:(e,o,r)=>{r.r(o),r.d(o,{assets:()=>l,contentTitle:()=>i,default:()=>p,frontMatter:()=>s,metadata:()=>c,toc:()=>a});var t=r(6070),n=r(1582);const s={title:"Artificial Intelligence",lang:"en-US"},i=void 0,c={id:"build-on-morph/developer-resources/use-ecosystem-developer-tools/artificial-intelligence",title:"Artificial Intelligence",description:"ORA",source:"@site/docs/build-on-morph/developer-resources/use-ecosystem-developer-tools/7-artificial-intelligence.md",sourceDirName:"build-on-morph/developer-resources/use-ecosystem-developer-tools",slug:"/build-on-morph/developer-resources/use-ecosystem-developer-tools/artificial-intelligence",permalink:"/docs/build-on-morph/developer-resources/use-ecosystem-developer-tools/artificial-intelligence",draft:!1,unlisted:!1,editUrl:"https://github.com/morph-l2/morph-doc/tree/main/docs/build-on-morph/developer-resources/use-ecosystem-developer-tools/7-artificial-intelligence.md",tags:[],version:"current",sidebarPosition:7,frontMatter:{title:"Artificial Intelligence",lang:"en-US"},sidebar:"DeveloperSidebar",previous:{title:"Block Explorer",permalink:"/docs/build-on-morph/developer-resources/use-ecosystem-developer-tools/block-explorer"},next:{title:"Decentralized Naming Services",permalink:"/docs/build-on-morph/developer-resources/use-ecosystem-developer-tools/decentralized-naming-services"}},l={},a=[{value:"ORA",id:"ora",level:3}];function d(e){const o={a:"a",h3:"h3",img:"img",p:"p",...(0,n.R)(),...e.components};return(0,t.jsxs)(t.Fragment,{children:[(0,t.jsx)(o.h3,{id:"ora",children:"ORA"}),"\n",(0,t.jsx)(o.p,{children:(0,t.jsx)(o.img,{alt:"ORA",src:r(1078).A+"",width:"2880",height:"1476"})}),"\n",(0,t.jsx)(o.p,{children:"ORA provides chain-agnostic infrastructure that bridges the gap between AI and blockchain."}),"\n",(0,t.jsx)(o.p,{children:"ORA empower developers with the tools necessary to build end-to-end trustless and decentralized applications enhanced by verifiable AI."}),"\n",(0,t.jsxs)(o.p,{children:["You can find the contract addresses to our AI Oracle on Morph ",(0,t.jsx)(o.a,{href:"https://docs.ora.io/doc/ai-oracle/ai-oracle/references#morph-mainnet",children:"here"})]})]})}function p(e={}){const{wrapper:o}={...(0,n.R)(),...e.components};return o?(0,t.jsx)(o,{...e,children:(0,t.jsx)(d,{...e})}):d(e)}},1078:(e,o,r)=>{r.d(o,{A:()=>t});const t=r.p+"assets/images/biconomy-837b6cf4a7c94f3bf1da46a565cc25e6.png"},1582:(e,o,r)=>{r.d(o,{R:()=>i,x:()=>c});var t=r(758);const n={},s=t.createContext(n);function i(e){const o=t.useContext(s);return t.useMemo((function(){return"function"==typeof e?e(o):{...o,...e}}),[o,e])}function c(e){let o;return o=e.disableParentContext?"function"==typeof e.components?e.components(n):e.components||n:i(e.components),t.createElement(s.Provider,{value:o},e.children)}}}]);
```

Biome (2.1.2)

[error] This hook is being called indirectly and conditionally, but all hooks must be called in the exact same order in every component render.

This is the call path until the hook.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)